.github/workflows/prepare-release.yml

```
@@ -281,12 +300,18 @@ jobs:
           echo "$RELEASE_NOTES" >> $GITHUB_ENV
           echo "EOF" >> $GITHUB_ENV
 
-      - name: Upload Standalone Artifacts
+      - name: Upload Standalone artifacts
```

Nit: let's change all or nothing :P
```suggestion
      - name: Upload standalone artifacts
```